ownCloudAppShared/Branding/Branding+App.swift

```diff
@@ -211,7 +214,23 @@ extension Branding : BrandingInitialization {
 				.description    : "Indicates if the user can change the server URL for the account.",
 				.status        : OCClassSettingsKeyStatus.advanced,
 				.category    : "Branding"
-			]
+			],
+            
+            .sidebarLinks : [
+                .type         : OCClassSettingsMetadataType.array,
+                .label         : "Sidebar Links",
+                .description    : "Array with Links, which should appear in the sidebar.",
```

Comment:
The description suggests that the parameter expects an array of URLs, whereas it expects an array of dictionaries where the keys `url` and `title` are mandatory - and where an optional image can be added as either an SF-Symbol name (`symbol`) or the name of an image bundled with the app (`image`).

Reply:
fixed in the latest commit

Reply:
Thanks!